en/starter/gettingstarted.md

```diff
@@ -28,12 +34,20 @@ Installing Kitura's Homebrew tap will allow ```brew``` to access Kitura's packag
 brew tap ibm-swift/kitura
 ```
 
-<span class="arrow">&#8227;</span> Install Kitura's command-line interface.
+<span class="arrow">&#8227;</span> Install Kitura's CLI.
 
 ```
 brew install kitura
 ```
 
+### Linux
+
+```
+npm install -g kitura-cli
+```
+
+## Generating your first Kitura project
+
```

Comment:
@lroseblade and @seabaylea - are you happy with this proposal to change the Getting Started guide to now explicitly include instructions for Linux as well as macOS, as this will make the Getting Started guide which we streamlined to be the minimum size a fair bit longer? 

The section David has written above would need to be added in, then in addition further down this page we reference opening the project in Xcode and clicking the Run button in Xcode, these would also both need changes.